tests/toolchain/install.pm

```diff
@@ -47,10 +47,10 @@ sub run {
         script_run 'export CC=/usr/bin/gcc-5';
         script_run 'export CXX=/usr/bin/g++-5';
     }
-    elsif (is_sle) {
+    elsif (is_sle or is_tumbleweed or (is_leap and leap_version_at_least('15.0'))) {
```

Comment:
Again, asking for tumbleweed is a sign that something went wrong.

Reply:
what about caasp?

Reply:
The default case should be not be attached to a fixed version, so empty the condition should be fine. Not considering caasp, that could be the only exception that I found.